microsoft-365/enterprise/cross-tenant-mailbox-migration.md

```diff
@@ -580,7 +584,7 @@ However, there are some things you need to be aware of.
 
 - **Issue: Post migration Teams functionality in the source tenant will be limited.** After the mailbox is migrated to the target tenant, Teams in the source tenant will no longer have access to the user's mailbox. So, if a user logs into Teams with the source tenant credential, then there will be a loss of functionality such as the inability to update your profile picture, no calendar application, and an inability to search and join public teams.
 
-- **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive – meaning more than one archive mailbox, the feature is unable to migrate the additional archives and should fail.
+- **Issue: Auto Expanded archives cannot be migrated.** The cross-tenant migration feature supports migrations of the primary mailbox and archive mailbox for a specific user. If the user in the source however has an auto expanded archive enabled – meaning more than one archive mailbox (aux archives), the feature is able to migrate them as long as there are no more than 10 aux archives. If there are more than 10 aux archives, the migration will fail.
```

Comment:
We already have this posted in this document under Do auto-expanded archive mailboxes move?
Yes, if the user in source has auto-expanding archives enabled and has additional auxiliary archives, cross-tenant mailbox migration will work. We support moving users that have no more than 12 auxiliary archive mailboxes.

So we just need to remove this line completly

Reply:
done